src/main/java/com/KacFlor/ShopSpring/controller/UserController.java

```diff
@@ -23,9 +23,40 @@ public UserController(UserService userService){
         this.userService = userService;
     }
 
-    @GetMapping
-    public List<User> getUser(){
-        return this.userService.getUser();
+    @PreAuthorize("hasAuthority('" + Role.Fields.ADMIN + "')")
+    @DeleteMapping("/delete-id")
```

Comment:
Not like this, it schould be Delete "user/{id}"

Reply:
https://restfulapi.net/resource-naming/ !!!!!!!!!!

Reply:
Use nouns

Reply:
one